docs/UserGuide.md

```diff
@@ -303,10 +303,13 @@ Format: `addorder INDEX d/DATE r/REMARK`
 * The date must be in the format `YYYY-MM-DD`, where `YYYY` is the year (all the digits, i.e. 2012), `MM` is the month (01 to 12) and `DD` is the day (01 to 31). For example, `2020-12-31`.
 
 <box type="tip" seamless>
-
 **Note:** A person can have any number of orders (including 0)
 </box>
 
+<box type="tip" seamless>
+**Note:** You can add an order with the same remark and date to the same supplier multiple times. This is because we understand that you may want to make duplicate orders during festive seasons or special occasions.
```

Comment:
I know how this is supposed to address the comments made in the issue, but the reason can be more "general" and "convincing". For example, "This is because we understand that you may want to quickly make duplicate orders when demand is high without the hassle of deleting your original order and adding back the updated version again" Or something like that.